app/src/main/java/com/example/parkingsystem_kotlin/mvp/view/ParkingReservationView.kt

```diff
@@ -2,40 +2,58 @@ package com.example.parkingsystem_kotlin.mvp.view
 
 import android.app.Activity
 import android.widget.EditText
-import android.widget.Toast
 import com.example.parkingsystem_kotlin.R
 import com.example.parkingsystem_kotlin.databinding.ActivityParkingReservationBinding
 import com.example.parkingsystem_kotlin.mvp.contract.ParkingReservationContract
 import com.example.parkingsystem_kotlin.mvp.view.base.ActivityView
+import com.example.parkingsystem_kotlin.util.Constants
 import com.example.parkingsystem_kotlin.util.DateUtils.getInputTypeCalendar
 import com.example.parkingsystem_kotlin.util.DateUtils.showDatePickerDialog
+import com.example.parkingsystem_kotlin.util.MsgToShow
+import com.example.parkingsystem_kotlin.util.showToast
 import java.util.Calendar
 
 class ParkingReservationView(private val binding: ActivityParkingReservationBinding, activity: Activity) :
     ParkingReservationContract.View, ActivityView(activity) {
 
-    override fun showToastParkingReservationIsOverlapped() {
-        showToast(R.string.toast_parking_reservation_activity_date_is_overlapped)
-    }
-
-    override fun parkingReservationSuccess() {
-        showToast(R.string.toast_parking_reservation_activity_reservation_success)
-        cancelActivity()
+    override fun showToastParkingReservation(message: MsgToShow) {
```

Comment:
this logic should be in the presenter.. and the presenter call each correspondent view method